ROMFS/px4fmu_common/init.d/airframes/4100_tiltquadrotor

```diff
@@ -24,8 +24,7 @@
 
 . ${R}etc/init.d/rc.mc_defaults
 
-#Parameters here:
-param set LED_RGB_MAXBRT 8
+param set-default LED_RGB_MAXBRT 8
```

Comment:
These kinds of user preferences should probably be purged from anything other than complete RTF systems.